features/Addtocart.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Add to cart
+
+Background:Add to cart
+	Given the user in Homepage and select on the Product
+    And The user can be navigated to the product details page 
+
+
+    Scenario:Validate Empty Add to cart(Neagtive scenario)
+    When The user clicks on Add to Cart button
+    Then the user should see 2 error messages "This is a required field"
```

Comment:
@karthikeyan7010 Then the user should see "This is a required field" at X &Y i.e you need to specify where it'll throw error